repository/repository.go

```diff
@@ -221,22 +221,22 @@ cat /var/lib/rancher/rke2/server/token
+| Containerd      | [{{.ContainerdVersion}}](https://github.com/k3s-io/containerd/releases/tag/{{.ContainerdVersion}})                      |
+| Runc            | [{{.RuncVersion}}](https://github.com/opencontainers/runc/releases/tag/{{.RuncVersion}})                              |
+| CNI Plugins     | [{{.CNIPluginsVersion}}](https://github.com/containernetworking/plugins/releases/tag/{{.CNIPluginsVersion}})                      |
+| Metrics-server  | [{{.MetricsServerVersion}}](https://github.com/kubernetes-sigs/metrics-server/releases/tag/{{.MetricsServerVersion}})                   |
+| CoreDNS         | [{{.CoreDNSVersion}}](https://github.com/coredns/coredns/releases/tag/{{.CoreDNSVersion}})                                  |
+| Ingress-Nginx   | [{{.IngressNginxVersion}}](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-{{.IngressNginxVersion}})                                  |
+| Helm-controller | [{{.HelmControllerVersion}}](https://github.com/k3s-io/helm-controller/releases/tag/{{.HelmControllerVersion}})                         |
 
 ### Available CNIs
 | Component       | Version                                                                                                                                                                             | FIPS Compliant |
 | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
-| Canal (Default) | [Flannel v0.13.0-rancher1](https://github.com/k3s-io/flannel/releases/tag/v0.13.0-rancher1)<br/>[Calico v3.13.3](https://docs.projectcalico.org/archive/v3.13/release-notes/#v3133) | Yes            |
-| Calico          | [v3.19.2](https://docs.projectcalico.org/release-notes/#v3192)                                                                                                                      | No             |
-| Cilium          | [v1.9.8](https://github.com/cilium/cilium/releases/tag/v1.9.8)                                                                                                                      | No             |
-| Multus          | [v3.7.1](https://github.com/k8snetworkplumbingwg/multus-cni/releases/tag/v3.7.1)                                                                                                    | No             |
+| Canal (Default) | [Flannel {{.FlannelVersionRKE2}}](https://github.com/k3s-io/flannel/releases/tag/{{.FlannelVersionRKE2}})<br/>[Calico {{.CalicoVersion}}](https://projectcalico.docs.tigera.io/archive/v3.22/release-notes/#{{ .CalicoVersionTrimmed }}) | Yes            |
+| Calico          | [{{.CalicoVersion}}](https://projectcalico.docs.tigera.io/archive/v3.22/release-notes/#{{ .CalicoVersionTrimmed }})                                                                                                                      | No             |
```

Comment:
Can we template the `v3.22` portion of the URL as well? I guess that can be fixed up manually when the minor version is bumped?
```suggestion
| Calico          | [{{.CalicoVersion}}](https://projectcalico.docs.tigera.io/archive/v3.22/release-notes/#{{ .CalicoVersionTrimmed }})                                                                                                                      | No             |
```

Reply:
go template should fix that for us, but for some reason function like trimAll and replace do not work

Reply:
for example instead of CalicoVersionTrimmed it should be {{ trimAll .CalicoVersion "." }}